nix/services/redis_test.nix

```diff
@@ -1,6 +1,12 @@
 { pkgs, config, ... }: {
   services.redis."redis1".enable = true;
 
+  services.redis."redis2" = { config, ... }: {
+    enable = true;
+    port = 0;
+    unixSocket = "./redis.sock";
```

Comment:
What's the significance of `port = 0`? What happens if the user specifies both `port` and `unixSocket`?

Reply:
`port = 0` disables listening on the TCP port. Setting both is also allowed by redis, in which case it listens on both `unixSocket` and TCP `port`. Hence I just display a warning if both are set instead of error it out (https://github.com/juspay/services-flake/blob/821dbf382a5ff312aec99e4ded9da46a4eee21c3/nix/services/redis.nix#L28-L32).

Reply:
This should be documented then: #358

Reply:
It is documented in the option description. I will also add it to redis.md